spec/features/trainer_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+describe "training without cards" do
+  before :all do
+    Card.destroy_all
+    Card.create(original_text: 'дом', translated_text: 'house')
+  end
+
+  it 'no cards' do
+    visit '/'
+    expect(page).to have_content 'Ожидайте наступления даты пересмотра.'
+  end
+end
+
+describe "training with some cards" do
+  before :each do
+    Card.destroy_all
+    card = Card.create(original_text: 'дом', translated_text: 'house')
+    card.update_attribute(:review_date, Time.now)
+    card = Card.create(original_text: 'машина', translated_text: 'car')
+    card.update_attribute(:review_date, Time.now)
+  end
+
+  it 'first visit' do
+    visit '/'
+    expect(page).to have_content 'Original text'
+  end
+
+  it 'incorrect translation' do
+    visit '/'
+    fill_in 'user_translation', with: 'RoR'
+    click_button 'Проверить'
+    expect(page).
+        to have_content 'Вы ввели не верный перевод. Повторите попытку.'
```

Comment:
Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.